netci.groovy

```diff
@@ -641,7 +641,7 @@ def static setJobTimeout(newJob, isPR, architecture, configuration, scenario, is
             timeout = 360
         }
         else if (isJitStressScenario(scenario)) {
-            timeout = 240
+            timeout = 300
```

Comment:
nit maybe specify unit of `timeout` like `timeoutMin`

Reply:
We have larger refactoring of this file waiting. I would rather push this to a separate change.